src/serverrpc.cpp

```diff
@@ -226,37 +272,64 @@ CServerRpc::CServerRpc ( CServer* pServer, CRpcServer* pRpcServer, QObject* pare
     } );
 }
 
-QJsonValue CServerRpc::SerializeRegistrationStatus ( ESvrRegStatus eSvrRegStatus )
+const std::unordered_map<EDirectoryType, std::string> CServerRpc::sumDirectoryTypeToString = {
```

Comment:
Do we have a place where we already have those pre defined directories? Isn't it somewhere in util.h?

Reply:
There's a function to return the translated name.  This is a serialisation of the enum, though, so it must not use the translations.